discovery-provider/default_config.ini

```diff
@@ -17,6 +17,7 @@ trending_refresh_seconds = 3600
 infra_setup =
 indexing_transaction_index_sort_order_start_block =
 get_users_cnode_ttl_sec = 5
+backfill_stop_sig = 
```

Comment:
is there a single backfill stop signature, or is it different for each indexer?

Reply:
Just pushed changes that'll lookup a sig for each indexer individually. But fwiw I'm pretty sure I observed it working with having the same sig for all 3.